swagger/v1/component_schemas/ECFParticipantDeclarationPost2024ECTRetainedAttributesRequest.yml

```diff
@@ -0,0 +1,47 @@
+description: "An ECF participant retained declaration"
+type: object
+required:
+  - participant_id
+  - declaration_type
+  - declaration_date
+  - course_identifier
+  - evidence_held
+additionalProperties: false
+properties:
+  participant_id:
+    description: The unique ID of the participant
+    type: string
+    format: uuid
+    example: 3452b1a6-cbaa-422f-9ca9-40afa28583a2
+  declaration_type:
+    description: The event declaration type
+    type: string
+    enum:
+      - retained-1
+      - retained-2
+      - retained-3
+      - retained-4
+    example: retained-1
+  declaration_date:
+    description: The event declaration date
+    type: string
+    format: date-time
+    example: "2021-05-31T02:21:32.000Z"
+  course_identifier:
+    description: The type of course the participant is enrolled in
+    type: string
+    enum:
+      - ecf-induction
+    example: ecf-induction
+  evidence_held:
+    description: "The type of evidence that the lead provider holds on their platform to demonstrate that the participant has met the retention criteria for the current milestone period"
+    type: string
+    enum:
+      - training-event-attended
+      - self-study-material-completed
+      - materials-engaged-with-offline
+      - other
+      - "<br>For retained-2 declarations, providers will need to confirm if the engagement threshold has been reached:"
```

Comment:
discussed in person: we can have an enum of all values, and we move this explanation to the description, and include the list there as well, please check with Mark:
```
The type of evidence that the lead provider holds on their platform to demonstrate that the participant has met the retention criteria for the current milestone period. For retained-2 declarations, providers will need to confirm if the engagement threshold has been reached, and only accept those possible values: 75-percent-engagement-met, 75-percent-engagement-met-reduced-induction
```